### PR TITLE
Tuned rbd suites and confs

### PR DESCRIPTION
**Pull request by VasishtaShastry**
1) Fixed information in a suite
2) Removed all mds configuration on rbd mirroring confs and suites.
3) Added tier-2 conf and suite for rbd-mirroring
conf/pacific/rbd/tier-2_rbd_mirror.yaml
suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml

Changes made to files:
	modified:   conf/nautilus/rbd/tier-1_rbd_mirror.yaml
	modified:   suites/nautilus/rbd/tier-1_rbd_mirror.yaml
        modified:   suites/pacific/rbd/tier-0_rbd.yaml
	modified:   suites/pacific/rbd/tier-1_rbd_mirror.yaml
	modified:   suites/pacific/rbd/tier-1_rbd.yaml
	modified:   suites/pacific/rbd/tier-2_rbd_cli_regression.yaml
	modified:   suites/pacific/rbd/tier-2_rbd_regression.yaml
        added : suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml
        added :  conf/pacific/rbd/tier-2_rbd_mirror.yaml

Signed-off-by: Vasishta <vashastr@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by VasishtaShastry** [comment]
(New suite) suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml 
Logs (Jenkins) - xxxx.92.24/job/Custom Test Run/66/

**Comment by VasishtaShastry** [comment]
suites/nautilus/rbd/tier-1_rbd_mirror.yaml - .24/job/Custom Test Run/68

**Comment by VasishtaShastry** [comment]
suites/pacific/rbd/tier-1_rbd_mirror.yaml - xxxx.92.24/job/Custom Test Run/72

File: suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      clusters:
+        ceph-rbd1:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml:34]: Could we remove this key/value `registry-url`? Generally a key provided here is considered as an override to the defaults. As we would like to be environment agnostic, let's remove it.

File: suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                        - node5
+        ceph-rbd2:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    registry-url: registry.redhat.io
```
Comment (psathyan) [inline suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml:76]: Likewise.

**Review by psathyan — COMMENTED** [review]
Thanks for putting this together.

A minor request, can we rename the test suites to a more appropriate one? It would help with easy identifiaction.

**Comment by VasishtaShastry** [comment]
As changes made to tier-0 suite I've re-initiated 
xxxxx.24/job/Custom%20Test%20Run/74/

**Comment by VasishtaShastry** [comment]
@psathyan Is there anything else that I can work on from this PR's perspective?

**Review by psathyan — COMMENTED** [review]
Looks good to me

**Review by Manohar-Murthy — COMMENTED** [review]
lgtm, Team review is done.

**Comment by Manohar-Murthy** [comment]
We will have a dry run and merge.

**Comment by VasishtaShastry** [comment]
1) suites/pacific/rbd/tier-2_rbd_mirror_regression.yaml 
conf/pacific/rbd/tier-2_rbd_mirror.yaml
Logs (Jenkins) - xxxx.92.24/job/Custom Test Run/66/

2) suites/nautilus/rbd/tier-1_rbd_mirror.yaml
conf/nautilus/rbd/tier-1_rbd_mirror.yaml
.24/job/Custom Test Run/68

3) suites/pacific/rbd/tier-1_rbd_mirror.yaml - 
xxxx.92.24/job/Custom Test Run/72

4) suites/pacific/rbd/tier-0_rbd.yaml
xxxxx.24/job/Custom%20Test%20Run/74/

**Review by Manohar-Murthy — APPROVED** [review]

**Pull request merged**